apps/prairielearn/src/lib/db-types.ts

```diff
@@ -391,3 +391,92 @@ export const AssessmentQuestionSchema = z.object({
   tries_per_variant: z.number().nullable(),
 });
 export type AssessmentQuestion = z.infer<typeof AssessmentQuestionSchema>;
+
+export const RubricSchema = z.object({
+  created_at: DateFromISOString,
+  deleted_at: DateFromISOString.nullable(),
+  id: IdSchema,
+  max_extra_points: z.number(),
+  min_points: z.number(),
+  modified_at: DateFromISOString,
+  replace_auto_points: z.boolean(),
+  starting_points: z.number(),
+});
+export type Rubric = z.infer<typeof RubricSchema>;
+
+export const RubricItemSchema = z.object({
+  always_show_to_students: z.boolean(),
+  deleted_at: DateFromISOString.nullable(),
+  description: z.string().max(100),
+  explanation: z.string().max(10000).nullable(),
+  grader_note: z.string().max(10000).nullable(),
```

Comment:
As with column defaults, we shouldn't typically encode schema constraints (e.g. max values) in our Zod schemas since they won't actually do anything when parsing data from the database.